src/olympia/amo/tests/test_storage_utils.py

```diff
@@ -33,21 +32,22 @@ def test_storage_walk():
         results = [(dir, set(subdirs), set(files))
                    for dir, subdirs, files in sorted(walk_storage(tmp))]
 
-        yield (eq_, results.pop(0),
```

Comment:
I think the `yield`s were used to generate new test-cases based on what's in `results` but I don't think they added anything useful. Might be a good example of using `pytest.mark.parametrize` imho though.